sentry-android-core/src/main/java/io/sentry/android/core/cache/AndroidEnvelopeCache.java

```diff
@@ -132,9 +126,9 @@ public static boolean hasStartupCrashMarker(final @NotNull SentryOptions options
     final File crashMarkerFile = new File(outboxPath, STARTUP_CRASH_MARKER_FILE);
     try {
       final boolean exists =
-          options.getRuntimeManager().runWithRelaxedPolicy(() -> crashMarkerFile.exists());
+          options.getRuntimeManager().runWithRelaxedPolicy(crashMarkerFile::exists);
```

Comment:
We usually don't use method refs to maintain compatibility with AGP < 3.0 (although, frankly, we should've dropped it long time ago), so I'd suggest to revert this, and we'll convert all of them at once in a new major version of the SDK with a proper changelog entry.

Reply:
fixed in https://github.com/getsentry/sentry-java/pull/4933/commits/c602ec950be31ae88260b6dded1691e953b0512a